src/utils/htmlTemplates.js

```diff
@@ -26,9 +27,18 @@
     </body>
   </html>`;
 
+ export const apiDocsUri = (apiObj) => {
+   for (const uri of apiObj.uris) {
```

Comment:
Alternativ kan du erstatte for loop'en med  `return apiObj.uris.find(uri => uri.includes(config.apiDocPath))`